### DISCUSSION TITLE
Hello. Does the prefetch address generated by Pythia sequentially access L1D, L2, LLC, and DRAM? I mean, does the prefetch address generated by Pythia access the cache and main memory in a traditional manner?

### DISCUSSION THREAD

**Discussion by szuhzw** [CMU-SAFARI/Pythia#15] [Q&A]
Hello. Does the prefetch address generated by Pythia sequentially access L1D, L2, LLC, and DRAM? I mean, does the prefetch address generated by Pythia access the cache and main memory in a traditional manner? @rahulbera

**Comment by rahulbera** [comment]
Yes.
